cli/cli_test.go

```diff
@@ -145,6 +145,79 @@ func TestERNCommands(t *testing.T) {
 	}
 }
 
+// TestERNCommands tests running the 'meta ern convert' and
+// 'meta ern index' commands.
```

Comment:
Update the comment to reference `EIDR` instead of `ERN` (same throughout the body of the test).